app/scripts/components/stories/hub/index.tsx

```diff
@@ -161,150 +163,153 @@ function StoriesHub() {
     <PageMainContent>
       <LayoutProps
         title={getString('stories').other}
-        description={getString('storiesBanner')?.other || 'Explore the guided narratives below to discover how NASA satellites and other Earth observing resources reveal a changing planet.'}
-      />
-      <PageHero
-        title={getString('stories').other}
-        description={getString('storiesBanner')?.other || 'Explore the guided narratives below to discover how NASA satellites and other Earth observing resources reveal a changing planet.'}
+        description={getString('storiesBanner').other}
       />
-
+      <ContentOverride with='storiesHubHero'>
+        <PageHero
+          title={getString('stories').other}
+          description={getString('storiesBanner').other}
+        />
+      </ContentOverride>
```

Comment:
Seems that this should be a `ComponentOverride` which doesn't use frontmatter variables or markdown. Seems the [same that is being done on the homepage](https://github.com/NASA-IMPACT/veda-ui/blob/main/app/scripts/components/home/index.tsx#L137-L164), but not sure about the endgoal

Reply:
Yeah I think you are right, changed it to comonentOverride

Reply:
My suggestion was to just change this one to ComponentOverride. The other one should stay as ContentOverride as it replaces the whole page content.